File: VB/WindowsFormsApplication88/CustomColorPickEditPopupForm.vb
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Popup
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication88
    Public Class CustomColorPickEditPopupForm
        Inherits PopupColorPickEditForm

        Public Sub New(ByVal ownerEdit As CustomColorPickEdit)
            MyBase.New(ownerEdit)
        End Sub
        Protected Overrides Function CreatePopupColorEditBuilder() As PopupColorBuilder
            Return New CustomPopupColorBuilder(Me)
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication88/CustomColorPickEditPopupForm.cs
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Popup;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication88 {
    public class CustomColorPickEditPopupForm : PopupColorPickEditForm {
        public CustomColorPickEditPopupForm(CustomColorPickEdit ownerEdit) : base(ownerEdit) {
        }
        protected override PopupColorBuilder CreatePopupColorEditBuilder() {
            return new CustomPopupColorBuilder(this);
        }
    }
}
